### ISSUE TITLE
Swap out capsule with humanoid model

### ISSUE THREAD

**Issue by VTVDV** [DP94/RogueTrader#14]
Adding a humanoid model would let us use animations.

**Comment by DP94** [comment]
This could potentially be a big bit of work

**Comment by VTVDV** [comment]
Agreed, think when it comes to it we'll need to break this up into smaller issues to handle different aspects affected by this change (i.e ticket to change animations for things etc.).
